modules/collecting-gitops-debugging-data.adoc

```diff
@@ -27,7 +27,14 @@ Use the `oc adm must-gather` CLI command to collect the following details about
 +
 [source,terminal]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/gitops-must-gather-rhel8:v1.9.0
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:<image_version_tag> <1>
+----
+<1> The must-gather image for {gitops-shortname}.
++
+.Example command
+[source,terminal]
+----
+$ oc adm must-gather --image=registry.redhat.io/openshift-gitops-1/must-gather-rhel8:v1.9.0
 ----
```

Comment:
@varshab1210  and @reginapizza:  Question about **L46**:

Should we change **L46** to:

$ tar -cvaf must-gather.tar.gz must-gather.local.4157245944708210399 <1>
<1> Make sure to replace `must-gather-local.4157245944708210399/` with the actual directory name.

Reply:
Yes we can do that. I'll approve it either way

Reply:
@reginapizza Done. I have modified the content as discussed. Thanks for the approval.